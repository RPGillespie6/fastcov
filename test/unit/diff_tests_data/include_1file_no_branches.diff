diff --git a/test/functional/cmake_project/src/source1.cpp b/test/functional/cmake_project/src/source1.cpp
index 0c5409a..83c48bf 100644
--- a/test/functional/cmake_project/src/source1.cpp
+++ b/test/functional/cmake_project/src/source1.cpp
@@ -2,13 +2,13 @@
 
 int foo(bool take_branch)
 {
-    int x = 0;
-
+    int x = 0;//test
+//test
     for (int i = 0; i < 1000; i++) {
-        x += 2*i;
+        x += 2*i;//test
 
         if (take_branch)
-            x += i;
+            x += i;//test
     }
 
     return x;
