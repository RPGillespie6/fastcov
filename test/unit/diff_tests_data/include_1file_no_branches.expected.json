{
    "sources": {
        "/mnt/workspace/test/functional/cmake_project/src/source1.cpp": {
            "": {
                "functions": {},
                "branches": {},
                "lines": {
                    "5": 1,
                    "8": 1000,
                    "11": 0
                }
            }
        }
    }
}